app/views/layouts/govuk_template.html.erb

```diff
@@ -4,6 +4,7 @@
 
 <head>
   <title><%= yield(:page_title) %></title>
+  <%= csp_meta_tag %>
```

Comment:
Would this be best to go in the application layout in the content_for(:head) block?
On Apply we have this block (ignore the analytics line of course)

```ruby
<% content_for(:head) do %>
  <%= csrf_meta_tags %>
  <%= csp_meta_tag %>
  <%= render partial: 'layouts/analytics' if analytics_allowed? %>
<% end %>
```

Reply:
Oh good point, I forgot that!

Reply:
I'd quite like to have a simpler static "external" layout so perhaps something to do if that happens.